src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -72,11 +73,26 @@
     help="""Python version to use to create the virtual environment and
 run the Python file. Python3 is used by default in the cluster.""",
 )
+@click.option(
+    "--inputs",
+    default=None,
+    type=str,
+    help="""
+    Input arguments for the simulation. You can specify several arguments by joining them with commas, therefore, strings defined this way cannot contain commas.
+    PyMAPDL will try to convert, whenever possible, these inputs to ints or floats, otherwise, they remind as strings. These arguments can be changed in the HPS website. For example: --inputs="force=123,value='mystring'"
```

Comment:
```suggestion
    Input arguments for the simulation. You can specify several arguments by joining them with commas. Thus, strings defined in this way cannot contain commas.
    PyMAPDL converts these inputs to integer or float values when possible. Otherwise, they remain as strings. You can change these arguments on the HPS website. For example, ``--inputs="force=123,value='mystring'"``.
```